SYCL/ESIMD/dpas/dpas_fp16.cpp

```diff
@@ -0,0 +1,34 @@
+//==---------------- dpas_fp16.cpp  - DPC++ ESIMD on-device test ----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// This test verifies DPAS support for float16.
+
+#include "dpas_common.hpp"
+
+int main(int argc, const char *argv[]) {
+  queue Q(esimd_test::ESIMDSelector{}, esimd_test::createExceptionHandler());
+
+  bool Print = argc > 1 && std::string(argv[1]) == "-debug";
+  bool Passed = true;
+
+  // Test unsigned 2-bit integers./
```

Comment:
typo - 2-bit integer -> half float

Reply:
Will fix in next PR adding more tests. Thank you.

Reply:
Fixed here: https://github.com/intel/llvm-test-suite/pull/1291